FAQ.md

```diff
@@ -1,51 +1,100 @@
 - [user-callback and other PlugIns](#user-callback-and-other-plugins)
-  * [How to backup Debian/Ubuntu Package selection?](#how-to-backup-debianubuntu-package-selection)
-  * [How to restore Debian/Ubuntu Package selection?](#how-to-restore-debianubuntu-package-selection)
+   * [How to backup Debian/Ubuntu Package selection?](#how-to-backup-debianubuntu-package-selection)
+   * [How to restore Debian/Ubuntu Package selection?](#how-to-restore-debianubuntu-package-selection)
 - [Hardware-specific Setup](#hardware-specific-setup)
-  * [How to use QNAP QTS with BIT over SSH](#how-to-use-qnap-qts-with-bit-over-ssh)
-  * [How to use Synology DSM 5 with BIT over SSH](#how-to-use-synology-dsm-5-with-bit-over-ssh)
-  * [How to use Synology DSM 6 with BIT over SSH](#how-to-use-synology-dsm-6-with-bit-over-ssh)
-  * [How to use WesterDigital MyBook World Edition with BIT over ssh?](#how-to-use-westerdigital-mybook-world-edition-with-bit-over-ssh)
-- [Uncategorised questions](#uncategorised-questions)
-  * [Version >= 1.2.0 works very slow / Unchanged files are backed up](#version--120-works-very-slow--unchanged-files-are-backed-up)
-  * [Which additional features on top of a GUI does BIT provide over a self-configured rsync backup? I saw that it saves the names for uids and gids, so I assume it can restore correctly even if the ids change. Great! :-) Are there additional benefits?](#which-additional-features-on-top-of-a-gui-does-bit-provide-over-a-self-configured-rsync-backup-i-saw-that-it-saves-the-names-for-uids-and-gids-so-i-assume-it-can-restore-correctly-even-if-the-ids-change-great---are-there-additional-benefits)
-  * [How to move snapshots to a new hard-drive?](#how-to-move-snapshots-to-a-new-hard-drive)
-  * [How to move large directory in source without duplicating backup?](#how-to-move-large-directory-in-source-without-duplicating-backup)
+   * [How to use QNAP QTS NAS with BIT over SSH](#how-to-use-qnap-qts-nas-with-bit-over-ssh)
+   * [How to use Synology DSM 5 with BIT over SSH](#how-to-use-synology-dsm-5-with-bit-over-ssh)
+   * [How to use Synology DSM 6 with BIT over SSH](#how-to-use-synology-dsm-6-with-bit-over-ssh)
+   * [How to use Western Digital MyBook World Edition with BIT over ssh?](#how-to-use-western-digital-mybook-world-edition-with-bit-over-ssh)
+- [Uncategorized questions](#uncategorized-questions)
+   * [Version >= 1.2.0 works very slow / Unchanged files are backed up](#version--120-works-very-slow--unchanged-files-are-backed-up)
+   * [Which additional features on top of a GUI does BIT provide over a self-configured rsync backup? I saw that it saves the names for uids and gids, so I assume it can restore correctly even if the ids change. Great! :-) Are there additional benefits?](#which-additional-features-on-top-of-a-gui-does-bit-provide-over-a-self-configured-rsync-backup-i-saw-that-it-saves-the-names-for-uids-and-gids-so-i-assume-it-can-restore-correctly-even-if-the-ids-change-great-are-there-additional-benefits)
```

Comment:
Modify the URL. The emoticon is not correct interpreted by the toc generator

`if-the-ids-change-great-are-there-additional-benefits` -> `if-the-ids-change-great---are-there-additional-benefits`

Reply:
I am not able to point the comment to the correct line. Am I wrong? ;)
It is line 39